src/AsciiManager.cpp

```diff
@@ -166,6 +166,80 @@ void AsciiManager::CutChain()
     // to free it!
 }
 
+#pragma var_order(charWidth, i, string, text, guiString, padding_1, padding_2, padding_3)
+void AsciiManager::DrawStrings(void)
+{
+    i32 padding_1;
+    i32 padding_2;
+    i32 padding_3;
+    i32 i;
+    BOOL guiString;
+    f32 charWidth;
+    AsciiManagerString *string;
+    u8 *text;
+
+    guiString = TRUE;
+    string = this->strings;
+    this->vm0.flags.flag0 = 1;
+    this->vm0.flags.anchor |= AnmVmAnchor_TopLeft;
```

Comment:
```suggestion
    this->vm0.flags.anchor = AnmVmAnchor_TopLeft;
```

My bad 😅 

Sry ^^'